src/components/CryptoAddress/component.tsx

```diff
@@ -39,59 +44,72 @@ export const Component = ({
     );
   }, [value]);
 
-  const scanQrCodeButton = (
-    <Button
-      variant="secondary"
-      shape="square"
-      size="increased"
-      onClick={() => setShowQRCode((value) => !value)}
-      data-testid={buildTestId('btn-scan-qr-code')}
-    >
-      <Icon.QRCode />
-    </Button>
+  const scanQrCodeButton = useMemo(
```

Comment:
One thing I was curious about is here - it seems that it is not a big computation so I was thinking whether we can just use general React component. It doesn't seem a big issue here though.